react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import AKButton from '@atlaskit/button';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog failure.
+ *
+ * @param {Object} props - The properties for the inline dialog.
+ * @returns {XML}
```

Comment:
I'll list some general style nits, knowing this is still work in progress:
- A Component's render method returns a React element not XML. The constructor would return a react component. 
- I think @atlastkit dependencies go before 'react'
- "extends Component" is more common than "extends React.Component"
- alpha order methods, exports, imports, etc, etc